server/app.ts

```diff
@@ -31,10 +32,14 @@ export default function createApp(userService: UserService): express.Application
   app.use(setUpStaticResources())
   nunjucksSetup(app, path)
   app.use(setUpAuthentication())
-  app.use(authorisationMiddleware())
 
-  app.use('/', indexRoutes(standardRouter(userService)))
+  // CSRF protection
+  if (!testMode) {
+    app.use(csurf())
+  }
 
+  app.use(authorisationMiddleware)
+  app.use(setupRoutes(services))
   app.use((req, res, next) => next(createError(404, 'Not found')))
```

Comment:
Replaces all application route setup with a single call to `setupRoutes(services)`